src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -181,6 +182,22 @@ trait BackendCall {
       Future.successful(ExecutionHash("", None))
   }
 
+  def pollingInitialInterval: FiniteDuration
+  def pollingMaxInterval: FiniteDuration
+  def pollingMultiplier: Double
+  def pollingInitialGap: FiniteDuration
```

Comment:
I'd be careful about making default values just because some of our current backends define them.

IMO traits should generally *not* have defaults anyway and when they do it's because you view it as being generally true for any implementor. Having it be true for what we have now isn't going to hold up in a world where there are a dozen backends but it has to remain in the API